src/devices/tuya.ts

```diff
@@ -25,6 +25,266 @@ const fzZosung = zosung.fzZosung;
 const tzZosung = zosung.tzZosung;
 const ez = zosung.presetsZosung;
 
+const PJ1203A = {
```

Comment:
Could you move this to `src/lib/tuya.ts`

Reply:
Are you sure? I was under the impression that  `src/lib/tuya.ts` was for generic and potentially reusable things.
Everything in `PJ1203A` is really specific to a single device.

Reply:
I have been pondering on that for a while and I have a proposal to better integrate `PJ1203A` in `devices/tuya.ts`. 

1. move `PJ1203A.sync_time_increase_seq` and `PJ1203A.strict_fz_datapoints` to `fzLocal` with a `PJ1203A_` prefix.
2. get rid of `PJ1203A.options` by declaring the two options in the device `Definition`.
    - or by putting them in a new local variable `optLocal` similar to `fzLocal` and `tzLocal`.
3. Move the content of `PJ1203A.valueConverters` into a new local variable `convLocal` 
4. Create a new local variable `storeLocal` to hold functions that provide a globalStore for a device and move  `PJ1203A.get_private_state` to `storeLocal.PJ1203A`.  
5 And finally, transform everything remaining in `PJ1203A` into function members of the private state (so within `storeLocal.PJ1203A`). That should probably be done anyway.

Reply:
Sounds good, can you commit the code so that I can properly review it?